browser/brave_wallet/sources.gni

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_browser_brave_wallet_sources = [
+  "//brave/browser/brave_wallet/brave_wallet_tab_helper.cc",
+  "//brave/browser/brave_wallet/brave_wallet_tab_helper.h",
+]
+
+if (!is_android) {
```

Comment:
do not create new sources.gni files. There is no reason for new code to have circular dependency issues